src/test/kotlin/no/nav/familie/ef/sak/api/gui/VedtakControllerTest.kt

```diff
@@ -81,6 +86,49 @@ internal class VedtakControllerTest : OppslagSpringRunnerTest() {
         validerBehandlingFatterVedtak()
     }
 
+    @Test
+    internal fun `skal kaste feil ved innvilgelse hvis vilkårsvurderinger mangler`() {
+        val behandlingId = opprettBehandling(vedtakResultatType = ResultatType.INNVILGE)
+        lagVilkårsvurderinger(behandlingId, ikkeLag = 1)
+        val response = sendTilBeslutterMedResponse(SAKSBEHANDLER)
+        val forventetFeilmelding = "Kan ikke innvilge hvis ikke alle vilkår er oppfylt for behandlingId: $behandlingId"
+        assertThat(response.body.frontendFeilmelding).isEqualTo(forventetFeilmelding)
+    }
+
+
+    @Test
+    internal fun `skal kaste feil ved innvilgelse hvis en ikke er innvilget`() {
+        val behandlingId = opprettBehandling(vedtakResultatType = ResultatType.INNVILGE)
+        lagVilkårsvurderinger(behandlingId, Vilkårsresultat.IKKE_OPPFYLT)
+        // Når
+        val response = sendTilBeslutterMedResponse(SAKSBEHANDLER)
+        // Da skal
+        val forventetFeilmelding = "Kan ikke innvilge hvis ikke alle vilkår er oppfylt for behandlingId: $behandlingId"
+        assertThat(response.body.frontendFeilmelding).isEqualTo(forventetFeilmelding)
```

Comment:
Her går det å bruke metode som allerede finnes
```suggestion
        sendTilBeslutter(SAKSBEHANDLER) { response ->
            assertThat(response.body.frontendFeilmelding)
                    .isEqualTo("Kan ikke innvilge hvis ikke alle vilkår er oppfylt for behandlingId: $behandlingId")
        }
```